src/templates/PoolV2/Activity/index.tsx

```diff
@@ -51,25 +56,12 @@ type Activity = {
   amount: string[]
 }
 
-export type ActivityCardProps = {
-  key: string
-  actionType: actionsType
-  date: Date
-  wallet: string
-  txHash: string
-  activityInfo: ActivityInfo[]
-  newBalancePool?: ActivityInfo[]
-  sharesRedeemed?: {
-    amount: string
-    value: string
-  }
-}
-
 const first = 10
 const Activity = () => {
-  const [optionsSelected, setOptionsSelected] = React.useState<Array<string>>(
-    []
-  )
+  const [optionsSelected, setOptionsSelected] =
+    React.useState<string[]>(initialFilterOptions)
```

Comment:
Remove not needed type argument
```suggestion
    React.useState(initialFilterOptions)
```